.github/workflows/send-screenshots-to-pixeleagle.yml

```diff
@@ -16,18 +16,29 @@ on:
         required: true
         type: string
 
+env:
+  # Unfortunately, we can't check secrets in `if:` conditionals. However, Github's own documentation
+  # suggests a workaround: Putting the secret in an environment variable, and checking that instead.
+  PIXELEAGLE_TOKEN_EXISTS: ${{ secrets.PIXELEAGLE_TOKEN != '' }}
+
 jobs:
   send-to-pixel-eagle:
     name: Send screenshots to Pixel Eagle
     runs-on: ubuntu-24.04
     steps:
+      - name: Notify user on non-existent token
+        if: ${{ ! fromJSON(env.PIXELEAGLE_TOKEN_EXISTS) }}
```

Comment:
In case you're curious:

* Environment variables are stored as strings. However, the strings `'true'` and `'false'` are valid JSON, and return booleans when converted to JSON.
* I wish I could put this `if:` conditional on the job itself - but those don't allow access to environment variables for some reason.